roles/b2bua/handlers/main.yml

```diff
@@ -1,6 +1,11 @@
 - name: reload asterisk dialplan
   command: asterisk -rx 'dialplan reload'
+  when: runtime
+
 - name: reload asterisk manager
   command: asterisk -rx 'manager reload'
+  when: runtime
+
 - name: reload asterisk
   command: asterisk -rx 'core reload'
+  when: runtime
```

Comment:
Thinking about the b2bua container, can we start those services independantly ? I means starting 3 containers with a "start dialplan" / "start manager" / "start core" respectively ? Or are they tied ?

Reply:
that's not services afaik that's commands for asterisk.